### PR TITLE
#465 added hibernate validators as dependency

### PR DESCRIPTION
**Pull request by krzyk**
#465
* recreated problem with IT test
* added hibernate validator as dependency

### CODE REVIEW

**Comment by davvd** [comment]
@krzyk Thanks, someone will review your pull request soon

**Comment by davvd** [comment]
@pinaf this pull request is for you, please review

File: qulice-maven-plugin/src/it/hibernate-validator-check/verify.groovy [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @version $Id$
+ *
+ * Validate that the build really failed and violations were reported.
+ */
+
+def log = new File(basedir, 'build.log')
+assert !log.text.contains(
+    'JSR-303 validator failed to initialize: Unable to create a Configuration, because no ' +
```
Comment (pinaf) [inline qulice-maven-plugin/src/it/hibernate-validator-check/verify.groovy]: @krzyk This message seems too restrictive and it doesn't match the error reported by @darkled . I suggest we go with `JSR-303 validator failed to initialize`.

**Comment by pinaf** [comment]
@krzyk 1 comment

**Comment by krzyk** [comment]
@pinaf done

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@pinaf paid, thanks, added **17 mins** to your account, payment `AP-87H85388WT463124V`, 47 hours and 16 mins was spent... review comments (c=2) added as a bonus... +17 to your rating, your total score is [+3946](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor deploy

**Pull request merged**